### ISSUE TITLE
[Research] General solution for focus-visible

### ISSUE THREAD

**Issue by RasmusKjeldgaard** [kirbydesign/designsystem#2034]
- [x] I have written a [good issue](https://github.com/kirbydesign/designsystem/wiki/The-Good%3A-Issue) 

## Describe the enhancement
It should be investigated how focus outline on components can be implemented, across all the relevant components in Kirby. 

The aim should be a general solution if possible, otherwise a set of strategies for different types of components. 
This should be implemented in #1863.

<hr />

## Checklist:

The following tasks should be carried out in sequence in order to follow [the process of contributing](https://github.com/kirbydesign/designsystem/blob/master/.github/CONTRIBUTING.md/#the-process-of-contributing) correctly.

### Refinement

- [ ] Request that the issue is [UX refined](https://github.com/kirbydesign/designsystem/blob/master/.github/CONTRIBUTING.md/#ux-refinement); do not proceed until this is done.
- [ ] Request that the issue is [tech refined](https://github.com/kirbydesign/designsystem/blob/master/.github/CONTRIBUTING.md/#tech-refinement); do not proceed until this is done.

**Comment by RasmusKjeldgaard** [comment]
The `:focus-visible` pseudo-class seems like a great candidate for applying the focus ring to suitable components, especially now that it will be supported in safari. This way, the focus indicator will only show when focussed by a keyboard.

Suitable components seem to be: 
- [ ] Button
- [ ] Calendar
- [ ] Card
- [ ] Checkbox (UA?)
- [ ] Dropdown
- [ ] Form Field (UA?)
- [ ] Link (Probably already handled by UA)
- [ ] Radio (UA?)
- [ ] Range
- [ ] Segment Control


Questions: 
Should the focus outline also appear on click (so more like focus, instead of focus-visible), or only keyboard navigation? 
What should the behavior of slide button, tabs, toggle, item and accordion be?
